test_unstructured_ingest/metrics/aggregate-scores-cct.tsv

```diff
@@ -1,3 +1,3 @@
 strategy	average	sample_sd	population_sd	count
-cct-accuracy	0.798	0.083	0.072	4

```

Comment:
@Klaijan I didn't change anything about the calculation of these values in my click wrapper refactor PR - is this change expected / acceptable?

Reply:
Yep should be ok! I updated the connectors (where to and not to copy files from) on Saturday and potentially did not run the fixture update.

Reply:
ok thanks for the update!